### PR TITLE
chore: Adds release notification action

### PR DESCRIPTION
**Pull request by mffap**
When a new release is published, then a webhook is sent to discord. Skip pre-releases.

### CODE REVIEW

File: .github/workflows/release-notification.yaml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+name: Release notification
+
+# Controls when the workflow will run
+on:
+  release:
+    types: [published]
+
+jobs:
+  test_message:
+    runs-on: ubuntu-latest
```
Comment (fforootd) [inline .github/workflows/release-notification.yaml:12]:
```suggestion
    runs-on: ubuntu-20.04
```

File: .github/workflows/release-notification.yaml [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+    steps:
+
+      # Get the fields for the release
+      - name: Get release
+        id: get_release
+        uses: bruceadams/get-release@v1.2.3
+        env:
+          GITHUB_TOKEN: ${{ github.token }}
+
```
Comment (fforootd) [inline .github/workflows/release-notification.yaml:25]:
```suggestion
      # Get the fields for the release
      - name: Get release
        id: get_release
        uses: bruceadams/get-release@v1.2.3
    
```

IMO not needed the release is public

**Comment by mffap** [comment]
@adlerhurst this might be an option to send "filtered" webhooks about new releases. What do you think?

**Review by adlerhurst — APPROVED** [review]

**Comment by adlerhurst** [comment]
> @adlerhurst this might be an option to send "filtered" webhooks about new releases. What do you think?

I think we should give it a try

**Pull request merged**